src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.logic.commands;
+
+import java.util.Collections;
+import java.util.Set;
+
+import com.google.common.collect.Sets;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.UnmodifiableObservableList;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.UniqueTagList;
+import seedu.address.model.task.ReadOnlyTask;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.UniqueTaskList;
+import seedu.address.model.task.UniqueTaskList.TaskNotFoundException;
+
+public class EditCommand extends Command {
+    public static final String COMMAND_WORD = "edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the task identified by the index number used in the last task listing.\n"
+            + "Parameters: INDEX (must be a positive integer) NAME | d/DESCRIPTION | e/TAG...\n"
+            + "Example: " + COMMAND_WORD + " 1 d/download How I Met Your Mother season 1" ;
+    public static final String MESSAGE_EDIT_SUCCESS = "Edit saved!";
+    public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in the address book";
```

Comment:
I see you have changed most of wording from address book to task force, but still there are remaining s of "address book" make sure to update them to match with the functionalities of your product